ouroboros-consensus-cardano/cddl/base.cddl

```diff
@@ -29,6 +29,7 @@ slotno = word64
 stake = rational
 
 withOrigin<v> = [] / [v]
+withOriginTH<v> = [0] / [1, v]
```

Comment:
Re-creating this because we didn't yet publish the consensus version that removes it.